### DISCUSSION TITLE
Netmiko Nokia _SROS

### DISCUSSION THREAD

**Discussion by magohil** [ktbyers/netmiko#3251] [Q&A]
Hi,

When execute command on Nokia MD-CLI it doesn't provide what command has been sent and prompt at bottom. Same work well for Classic-cli. Need to modify nokia_sros.py or something at router side ?

Example:
--------- **Classic-CLI**

***A:Hostname1# show version** 
TiMOS-B-22.4.R3 both/hops Nokia 7705 SAR Copyright (c) 2000-2022 Nokia.
All rights reserved. All use subject to applicable license agreements.
Built on Tue Aug 30 12:04:31 EDT 2022 by builder in /builds/F224B/R3/panos/main
***A:Hostname1# exit al**l 

--------- **MD-CLI** (No command in beginning and no end prompt)

TiMOS-C-22.7.R2 cpm/hops64 Nokia 7750 SR Copyright (c) 2000-2022 Nokia.
All rights reserved. All use subject to applicable license agreements.
Built on Fri Sep 16 10:20:29 PDT 2022 by builder in /builds/c/227B/R2/panos/main/sros

**Comment by ktbyers** [comment]
Netmiko should automatically strip the command and should automatically strip the trailing prompt. So the `MD-CLI` is the correct Netmiko behavior (and the classical-CLI behavior is incorrect).

Can you show me your code that is generating this output?

**Reply by magohil** [reply]
Thanks Kirk,
Understood. This behaviour create problem when we store command output to some text files. When someone else see that file they have no idea what device this output belong to and what command was triggered. For this I have applied session_log

Following is code:

import netmiko
from netmiko import ConnectHandler
import io
import os

current_dir = os.getcwd()
folder_path = os.path.join(current_dir, 'output')
ip_addr = 'x.x.x.x'
host = 'hostname1.txt'
file_path = os.path.join(folder_path, host)

alk = {
    'device_type': 'nokia_sros',
    'ip': ip_addr,
    'port': 22,
    'username': 'admin',
    'password': 'admin',
    "session_log": file_path
}

net_connect = ConnectHandler(**alk)
output = net_connect.send_command("show version",read_timeout=20)
print(output)
net_connect.disconnect()

**Reply by ktbyers** [reply]
Couldn't you just store the command that was run at the top of the file?

Something like this, but written to a file?

```python
cmd = "show version"
output = net_connect.send_command(cmd)
print(f"#### {cmd} ####")
print(output)
print("#" * 40)

Or another common practice is to make the file name correspond to the name of the command i.e. "show_version.txt".
